app/inputs/check_boxes_with_hints_input.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CheckBoxesWithHintsInput < Formtastic::Inputs::CheckBoxesInput
+  def choice_html(choice)
+    super(choice) <<
+    template.content_tag(:p, hint(choice), class: 'inline-hints')
+  end
+
+  private
+
+  def hint(choice)
+    options[:member_hint].call(choice.is_a?(Array) ? choice.last : choice)
```

Comment:
Why do we have it array sometimes, in case we don't need it as an array?

Reply:
I left it as it was, we don't need it because we're using this input only in one place so the choice is always the same format. I think it was done this way to follow formtastic implementation:
https://github.com/formtastic/formtastic/blob/24812e62ccc2bb8958acdedece276e204836c74b/lib/formtastic/inputs/base/choices.rb#L47-L56